src/main/java/mate/academy/internet/shop/controllers/GetOrderController.java

```diff
@@ -17,8 +17,12 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         String orderId = req.getParameter("order_id");
-        Order order = orderService.get(Long.valueOf(orderId));
-        req.setAttribute("order", order);
+        Order myOrder = orderService.get(Long.valueOf(orderId));
+        Long sum = myOrder.getProducts().stream()
+                .mapToLong(order -> order.getPrice().longValue())
+                .sum();
```

Comment:
Optional: create method for this in OrderService: public Long getOrderAmount(Order order);

Reply:
done